crates/bin/pd/Cargo.toml

```diff
@@ -28,7 +28,7 @@ anyhow                           = { workspace = true }
 ark-ff                           = { workspace = true, default-features = true }
 async-stream                     = { workspace = true }
 async-trait                      = { workspace = true }
-axum                             = "0.6"
+axum                             = { workspace = true, features = ["http2"] }
```

Comment:
I am not completely sure about axum, `axum::serve`, and when/how to activate `http2`. I have left a lengthy comment in `penumbra-core::rpc` to that effect.